app/controllers/friendships_controller.rb

```diff
@@ -10,7 +10,8 @@ def create
   end
 
   def update
-    friend = User.find(params[:user_id])
+    # friend = User.find(params[:user_id])
```

Comment:
Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.